packages/block-library/src/navigation-link/edit.js

```diff
@@ -315,17 +315,12 @@ export default function NavigationLinkEdit( {
 	 */
 	function removeLink() {
 		// Reset all attributes that comprise the link.
-		// It is critical that all attributes are reset
```

Comment:
I see that this was part of #46933. Was this a general fix or something critical only to Apply and Cancel button implementation?

Reply:
Hmmm. That's a good point. I'll have to take a closer look at this.

Reply:
I think just for Apply/Cancel.

Reply:
Thanks for looking into it.